pkg/draft/server.go

```diff
@@ -466,3 +509,56 @@ func complete(id, desc string, out chan<- *rpc.UpSummary, err *error) {
 func grpcError(err error) error {
 	return errors.New(grpc.ErrorDesc(err))
 }
+
+// tail writes up to limit number of lines of the file specified by path relative to the file end.
+func tail(buf *bytes.Buffer, limit int64, path string) error {
```

Comment:
I'm not sure what that buys us right now. Do you feel strongly about including that in this PR?

Reply:
It doesn't really add anything tangible in terms of features, but it does give more control over how the file is opened and with what permissions. For example, a unit test may wish to call [`os.TempFile`](https://golang.org/pkg/io/ioutil/#TempFile) and pass in the file handler to `tail` instead of the temporary file's absolute path, which then results in multiple file descriptors opening the same file.

I'm just pre-maturely optimizing at this point so feel free to ignore.